index.html

```diff
@@ -1,47 +1,161 @@
-<!DOCTYPE html>
```

Comment:
Please convert to UNIX style line endings like so: https://stackoverflow.com/questions/16768776/convert-line-endings

This way git won't think the entire file changed

Reply:
I did this, although because I changed the line endings, once again it thinks I changed the whole file 🤦‍♂